drivers/vci_mec172x.c

```diff
@@ -10,8 +10,13 @@
 #include "vci.h"
 #include "soc.h"
 
+#ifdef CONFIG_SOC_MEC172X_NSZ
 /* Set bits 0,1 and 2 to enable BGPO0, BGPO1 and BGPO2 */
 #define BGPO_EN_MASK	0x7U
+#else
+/* Set bits 0-4 to enable BGPO0-BGPO5 */
+#define BGPO_EN_MASK	0x1FU
```

Comment:
Shouldn't this be 0x3F for BGPO 0  to 5?
i assume one bit for each bgpo.

Reply:
Corrected